docs/guides/help-for-the-non-technical.md

```diff
@@ -0,0 +1,196 @@
+
+Redocly CLI allows you to manage complex API reference docs in a multi-file format, apply linting (validation) and quickly bundle them up for publishing. Redoc previews and publishes your API docs as straightforward, interactive documentation in a format that developers love. Both are highly configurable, and offer a range of config options.
+
+Redocly CLI and Redoc are known as CLI tools. The clue is in the name: *Redocly CLI*. CLI stands for Command-Line Interface.
+
+## Mastering the command line
+
+### What is a CLI?
+
+If you're a tech writer who is used to writing and publishing docs using traditional WYSIWYG tools, CLIs can seem scary at first, but they're actually pretty amazing. And bonus: you get to use a retro-looking terminal and pretend you've gone back in time to the 1980s!
+
+A CLI is a little terminal window that sends instructions (known as 'commands') to software and programs installed on your computer. There are many terminals out there. For Redocly CLI and Redoc, we use the `node.js` terminal to send commands.
+
+### What is a command?
+
+A command is exactly that: a written instruction sent to a computer program via a terminal window to command it to take some action.
+
+In the world of Redocly CLI and Redoc, commands are all about API definition files so they revolve around checking them (`lint`), splitting them up for easy maintenance (`split`) and putting them back together for publishing (`bundle`).
+
+Here are some examples of commands that you might type into a terminal window in order to send the Redocly CLI progam some instructions:
+
+`redocly lint`
+`redocly split`
+`redocly bundle`
+
+Commands are really flexible. You might want to `lint` the heck out of all your definitions or just one you're particularly suspicious of. You might decide to set rules that limit the errors returned. Or you may want to relax the rules for some definitions, and crack the whip on others. You do all of this simply by adding more parameters to the command:
+
+`redocly lint mydefinitions/myapi.yaml`
+`redocly lint --max-problems=25`
+`redocly split mydefinitions/myotherapi.yaml`
+
+You can even make errors returned by commands look pretty:
+
+`redocly lint --format=stylish`
+
+You'll learn how to [run a command](#how-to-run-a-command) soon. But first, let's get Redocly CLI and Redoc installed.
+
+## Installing Redocly CLI and Redoc
+Did you notice our wonderful [Quickstart guide](./docs/quickstart.md)? Did you read it and understand some of it, but not quite enough to install the software? That's OK! The guide was written primarily for developers so we're going to break it down a little more, just for you.
+
+### node.js
+The `node.js` terminal is a self-contained environment that works best when sending commands to Redocly CLI and Redoc. You need to install it on your computer.
+
+### GitHub (aka Git)
```

Comment:
I don't think we can put it as aka Git because Git is an open source project which GitHub, GitLab, and other competitors use in combination with their own nifty features.